src/svg.rs

```diff
@@ -28,32 +29,49 @@ impl From<io::Error> for Error {
 
 /// Rendered SVG.
 #[derive(Debug)]
-pub struct Svg {
+pub struct RenderedSvg {
     pub data: Vec<u8>,
     pub width: usize,
 }
 
+/// Parsed SVG.
+pub struct Svg {
+    tree: resvg::Tree,
+}
```

Comment:
Hm I'm wondering if we could combine the two structs to basically have "on-demand" rendering?

So you'd first construct an svg using the parse stuff, then you have a method to re-render/resize the SVG which is automatically cached.

The goal would be to just avoid having to store two separate structs.

Reply:
The issue with that approach is that the cached bitmap must still be invalidated on size change, and we might also want to keep only the parsed tree and not render it right away, for instance in a future PR I will split the .desktop into the same two steps, as we want to do it in parallel to the Wayland initialisation dance, but we don’t know at which size the final render will happen.

Reply:
`Option<Vec<u8>>` should handle all of that no problem, right?

Reply:
We can even have a `Vec<u8>` with the `.is_empty()` condition, yeah, I’ve merged them both here.